include/boost/winasio/http/http_initializer.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef BOOST_WINASIO_HTTP_INITIALIZER_HPP
+#define BOOST_WINASIO_HTTP_INITIALIZER_HPP
+
```

Comment:
Usually #pragma once is also added here inside the #define

Reply:
Ah.
```
#if defined(_MSC_VER) && (_MSC_VER >= 1200)
# pragma once
#endif // defined(_MSC_VER) && (_MSC_VER >= 1200)
```